gateway/src/cron.rs

```diff
@@ -131,11 +140,13 @@ impl CronSubmission {
         &mut self,
         store: &BS,
         submitter: Address,
+        submitter_weight: TokenAmount,
         checkpoint: CronCheckpoint,
-    ) -> anyhow::Result<u16> {
+    ) -> anyhow::Result<TokenAmount> {
         self.update_submitters(store, submitter)?;
+        self.total_submission_weight += submitter_weight.clone();
```

Comment:
Not a blocker (and I may be wrong) but I think that if you borrow here you save having to clone `submitter_weight`.
```suggestion
        self.total_submission_weight += &submitter_weight;
```

Reply:
yep, it works. I will update.